### PR TITLE
Add more clarity to log message when connection fails while downloading file

### PR DESCRIPTION
**Pull request by felipebrahm**
### Proposed changes in this pull request:
- Change the log message to include a better description of the actual error when trying to download a file
- [x] PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review
- [x] Read [contribution guide](https://github.com/SickRage/SickRage/blob/master/.github/CONTRIBUTING.md)
### Explanation

My SickRage setup had been working without an issue for a few months but all of a sudden all torrents failed to download with a log message like this:

```
Problem setting permissions or writing file to C:\Users\xxx\sickrage\foo.torrent
```

I tried changing the permissions of the folder but nothing helped. Then I modified the code so I could read the actual error message and this is what I got:

```
('Connection broken: IncompleteRead(29 bytes read)', IncompleteRead(29 bytes read))
```

Basically, it was failing to _download_ the file, not _write_ it. I still haven't been able to figure out why it's failing to download (all other scripts that are running on my computer are able to download files just fine), but in the meantime I'm submitting this PR to fix the log message.

### CODE REVIEW

**Comment by SickRage2** [comment]
Thanks for the pull request! Before a real human comes by, please make sure your PR has all of the below criteria checked
- [x] Give a description on what the PR is for.
- [x] Make sure your PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review

Please make sure you also read [contribution guide](https://github.com/SickRage/SickRage/blob/master/contributing.md) and followed all of the steps.

Thanks!

**Comment by felipebrahm** [comment]
@miigotu do you by any chance have any idea why I may be getting that _IncompleteRead_ error? I've been trying to debug it but I don't get it. SickRage had been running without any issues for a while and it suddenly stopped working (other scripts are working just fine).

**Comment by miigotu** [comment]
Don't know anything about your setup. Make sure python is 2.7.9+, otherwise check the url its coming from.
